api/lib/utils.js

```diff
@@ -125,6 +125,53 @@ function isValidLedgerWord(ledger) {
   return (/^current$|^closed$|^validated$/.test(ledger));
 }
 
+function getFee(remote) {
+  // https://wiki.ripple.com/Transaction_Fee#Base_Fees
+  var FEE_BASE_REFERENCE = 10;  // this is currently a constant in rippled
```

Comment:
Are you sure it's a constant? It might also be part of the configuration that we are running on our rippled servers